run_wml_tests

```diff
@@ -58,6 +58,9 @@ class TestListParser:
             test_list.append(t)
         return test_list
 
+def get_output_filename(args):
+    return "test-output-"+args[-1]
```

Comment:
While the last value of args is currently the name of a unit test, this seems fragile logic. I'd much prefer to have the caller pass a name for the log, even if that means the caller's caller has to pass another argument to the caller.

Also, if it's just a tempfile then would it be better to use a standard tempfile library?

Reply:
Updated to find it more intelligently.

On travis it's effectively a temporary file anyway, since it doesn't get preserved between builds, and if run locally the person running it may want to look at the output of any particular test run (whether it failed or not).